torch/testing/_internal/distributed/rpc/rpc_test.py

```diff
@@ -888,7 +889,7 @@ def test_stress_heavy_rpc(self):
 
     @dist_init
     def test_stress_heavy_rpc_torchscript(self):
-        self._stress_test_rpc(heavy_rpc_torchscript, repeat=20, args=(torch.ones(100, 100),))
+        self._stress_test_rpc(heavy_rpc_torchscript, repeat=20, args=())
```

Comment:
As discussed offline, there shouldn't be a race on the server side modifying the same tensor since the tensor is copied on the server side.